definitions.yml

```
@@ -716,5 +719,16 @@ definitions:
       frontend_url:
         type: string
         description: "The URL of the PN web frontend"
-
-
+  PaymentsConfig:
+    type: object
+    description: "Configuration data related to payments"
+    properties: 
+      allowedPspsByOrigin:
+       $ref: "#/definitions/AllowedPspsByOrigin"
+  AllowedPspsByOrigin:
+    type: object
+    properties: 
+      "poste_datamatrix_scan":
```

```suggestion
      "datamatrix":
```
maybe just `datamatrix` could be enough or _poste_ has its specific one? what do you think?

---

Yes it is a specific one for Poste. I am also adding scan to be consistent with the [origins](https://github.com/pagopa/io-app/blob/IA-927-datamatrix-psp/ts/store/actions/wallet/payment.ts#L40) defined in the app code.